pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -120,13 +121,13 @@
     "clickatell" = ps: with ps; [ ];
     "clicksend" = ps: with ps; [ ];
     "clicksend_tts" = ps: with ps; [ ];
+    "climacell" = ps: with ps; [ ]; # missing inputs: pyclimacell
     "climate" = ps: with ps; [ ];
     "cloud" = ps: with ps; [ aiohttp-cors hass-nabucasa ];
     "cloudflare" = ps: with ps; [ pycfdns ];
     "cmus" = ps: with ps; [ ]; # missing inputs: pycmus
     "co2signal" = ps: with ps; [ ]; # missing inputs: co2signal
     "coinbase" = ps: with ps; [ ]; # missing inputs: coinbase
-    "coinmarketcap" = ps: with ps; [ coinmarketcap ];
```

Comment:
Do we want to remove this python package from nixpkgs?

Reply:
Why? Because home-assistant is the only visible consumer?

Reply:
Yes and it is also maintainerless.

Reply:
https://github.com/home-assistant/core/pull/46615

Reply:
See https://github.com/NixOS/nixpkgs/pull/115050